specutils/io/default_loaders/tests/test_jwst_reader.py

```diff
@@ -366,7 +366,7 @@ def generate_s3d_wcs():
 
 
 @pytest.fixture()
-def tmp_asdf(tmpdir):
+def tmp_asdf():
```

Comment:
What hurts my head is `tmp_asdf` itself is a fixture but it took in `tmpdir` fixture. What was it trying to do?

Reply:
It is a mystery 😕